core/server/data/importer/importers/data/posts.js

```diff
@@ -4,13 +4,15 @@ const uuid = require('uuid');
 const BaseImporter = require('./base');
 const converters = require('../../../../lib/mobiledoc/converters');
 const validation = require('../../../validation');
+const postsMetaSchema = require('../../../schema').tables.posts_meta;
+const metaAttrs = _.keys(_.omit(postsMetaSchema, ['id']));
 
 class PostsImporter extends BaseImporter {
     constructor(allDataFromFile) {
         super(allDataFromFile, {
             modelName: 'Post',
             dataKeyToImport: 'posts',
-            requiredFromFile: ['posts', 'tags', 'posts_tags', 'posts_authors'],
+            requiredFromFile: ['posts', 'tags', 'posts_tags', 'posts_authors', 'posts_meta'],
```

Comment:
How does this behave when there are no records in `posts_meta` table? Just wondering about the situation where all posts don't have any records for `meta_title`, `meta_description` etc.

Reply:
Context: This reads value from imported json file and is used [here](https://github.com/TryGhost/Ghost/pull/11102/files#diff-e712df50c0dc7cf33746eeff0564003cR33). 

In case of v3 export, we will have `posts_meta: []` entry in the exported json, and in case of v2 exports, the `this.requiredFromFile.posts_meta` value will be `null`. In both cases, since we are using `_.find` from lodash, it will handle it and return `null` for `postsMetaFromFile`. 😄

Reply:
Awesome! No further Qs on this one :)